visualization/app/codeCharta/ui/artificialIntelligence/artificialIntelligence.component.html

```diff
@@ -3,33 +3,66 @@
 		<i class="fa fa-cogs"></i>
 		<span
 			id="ai-badge"
-			ng-show="!hideBadge && ($ctrl._viewModel.suspiciousMetricSuggestionLinks.length || $ctrl._viewModel.unsuspiciousMetrics.length)"
+			ng-show="
+				!hideBadge &&
+				($ctrl._viewModel.riskProfile !== undefined ||
+					$ctrl._viewModel.suspiciousMetricSuggestionLinks.length ||
+					$ctrl._viewModel.unsuspiciousMetrics.length)
+			"
 		>
-			{{ ($ctrl._viewModel.suspiciousMetricSuggestionLinks.length ? 1 : 0) + ($ctrl._viewModel.unsuspiciousMetrics.length ? 1 : 0) }}
+			{{
+				($ctrl._viewModel.riskProfile !== undefined ? 1 : 0) +
+					($ctrl._viewModel.suspiciousMetricSuggestionLinks.length ? 1 : 0) +
+					($ctrl._viewModel.unsuspiciousMetrics.length ? 1 : 0)
+			}}
 		</span>
 	</md-button>
 
 	<md-menu-content class="ai-drop-down" width="7">
-		<md-menu-item class="ai-item" ng-show="!$ctrl._viewModel.isFeatureApplicable">
-			<md-button ng-disabled="true">Suspicious Metrics can currently be found within JAVA maps only.</md-button>
-		</md-menu-item>
-
 		<md-menu-item class="ai-item" ng-show="$ctrl._viewModel.unsuspiciousMetrics.length"
-			><md-button ng-disabled="true">Unsuspicious Metrics</md-button></md-menu-item
+			><md-button ng-disabled="true">
```

Comment:
Gibt es an dieser Stelle einen Grund für den `<md-button>`? Wenn dieser sowieso permanent disabled ist, könnte auch ein `<div>` herhalten

Reply:
It is the standard for these menus, so I just adapted that way.
We should migrate AngularJS Compontents in the future, so that we can rework the whole menus.